src/main/java/shortestpath/pathfinder/Pathfinder.java

```diff
@@ -1,14 +1,8 @@
 package shortestpath.pathfinder;
 
 import java.time.Instant;
-import java.util.ArrayList;
-import java.util.Deque;
-import java.util.HashSet;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.PriorityQueue;
-import java.util.Queue;
-import java.util.Set;
+import java.util.*;
```

Comment:
It is a RuneLite convention to not use wild card imports, so I would prefer if you could list them as before.